jaas/src/main/java/com/griddynamics/jagger/jaas/rest/DataServiceRestController.java

```diff
@@ -1,161 +1,87 @@
 package com.griddynamics.jagger.jaas.rest;
 
-import com.griddynamics.jagger.engine.e1.services.DataService;
+import static com.griddynamics.jagger.jaas.service.DynamicDataService.DEFAULT_DB_CONFIG_ID;
+
 import com.griddynamics.jagger.engine.e1.services.data.service.MetricEntity;
 import com.griddynamics.jagger.engine.e1.services.data.service.MetricPlotPointEntity;
 import com.griddynamics.jagger.engine.e1.services.data.service.MetricSummaryValueEntity;
 import com.griddynamics.jagger.engine.e1.services.data.service.SessionEntity;
 import com.griddynamics.jagger.engine.e1.services.data.service.TestEntity;
 import com.griddynamics.jagger.jaas.service.DynamicDataService;
-import com.griddynamics.jagger.jaas.storage.model.DbConfigEntity;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.http.MediaType;
 import org.springframework.http.ResponseEntity;
-import org.springframework.web.bind.annotation.DeleteMapping;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.PathVariable;
-import org.springframework.web.bind.annotation.PostMapping;
-import org.springframework.web.bind.annotation.PutMapping;
-import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RequestParam;
 import org.springframework.web.bind.annotation.RestController;
-import org.springframework.web.servlet.support.ServletUriComponentsBuilder;
 
-import java.util.Arrays;
-import java.util.Collections;
 import java.util.List;
 import java.util.Map;
-import java.util.Optional;
 import java.util.Set;
-import java.util.function.Function;
 
 /**
- * JaaS REST API controller based on Spring MVC which exposes db backed resources.
+ * JaaS REST API controller based on Spring MVC which exposes DB backed resources
+ * of DB which is configured at deployment time
+ * in contrast to {@link DynamicDataServiceRestController}
+ * which is capable to register DB to connect to dynamically at runtime.
  */
-@RequestMapping(value = "/dbs")
+@RequestMapping(value = "/db")
 @RestController
 public class DataServiceRestController extends AbstractController {
```

Comment:
Shouldn't this class also have annotation @ConditionalOnProperty? If property is true - this one created, otherwise DynamicDataServiceRestController